src/test/java/uk/gov/hmcts/reform/sscs/service/NotificationServiceTest.java

```diff
@@ -2039,7 +2172,11 @@ private void verifyExpectedLogErrorCount(Appender<ILoggingEvent> mockAppender, A
         );
         List<ILoggingEvent> logEvents = (List<ILoggingEvent>) captorLoggingEvent.getAllValues();
         if (expectedErrors == 0) {
-            assertFalse(logEvents.stream().noneMatch(e -> e.getLevel().equals(Level.ERROR)));
+            if (logEvents.isEmpty()) {
+                assertEquals(logEvents.size(), expectedErrors);
```

Comment:
not sure if this change is needed. Isn't this the same as `assertFalse(logEvents.stream().noneMatch(e -> e.getLevel().equals(Level.ERROR)));` ?

Reply:
If logevents size is 0 then assertFalse(logEvents.stream().noneMatch(e -> e.getLevel().equals(Level.ERROR))) equals will not be called and condition won't meet

Reply:
It was failing test on this point